src/Symfony/Component/Messenger/Transport/Serialization/Serializer.php

```diff
@@ -30,6 +30,7 @@
  */
 class Serializer implements SerializerInterface
 {
+    const MESSENGER_SERIALIZATION_CONTEXT = 'MESSENGER_SERIALIZATION';
```

Comment:
should be `public const` and the value should probably be something lke `messenger_serialization`.